### ISSUE TITLE
tapViewWithAccessibilityLabel: and waitForViewWithAccessibilityLabel: fail for Back buttons

### ISSUE THREAD

**Issue by docmisterio** [kif-framework/KIF#529]
With KIF 3.1.0, we no longer are able to call tapViewWithAccessibilityLabel: or waitForViewWithAccessibilityLabel: with the label @"Back" for the back button on a UINavigationController. KIF complains that the view is hidden. Rolling back the added test for "alpha == 0.0" in viewContainingAccessibilityElement:tappable:error seems to make it work again.
